keps/sig-auth/3257-cluster-trust-bundles/README.md

```diff
@@ -290,7 +290,7 @@ kind: ClusterTrustBundle
 metadata:
   name: example.com:server-tls:foo
   labels:
-    kubernetes.io/cluster-trust-bundle-version: live
+    k8s.example/cluster-trust-bundle-version: live
```

Comment:
nit: I'd suggest sticking with example.com as the label namespace, consistent with the signer name domain

```suggestion
    example.com/cluster-trust-bundle-version: live
```

Reply:
As it is just an example, we can make this change, but as it is noticed as an k8s label, should we add the example.com instead of k8s.example? @sftim , WDYT? 
Thanks for the review @liggitt

Reply:
https://github.com/kubernetes/enhancements/pull/4971#discussion_r1851998358 was a nit, but it makes sense to me.

(no need to check with docs folks on this - the logic about the change was to avoid implying there's a known label key)

Label keys aren't API, but we still like to note which ones are recognized and which aren't.

Reply:
> Label keys aren't API, but we still like to note which ones are recognized and which aren't.

Is this for labels set by users? I thought those were part of the Kubernetes API.

Even if a label is only set by a Kubernetes component, it might still be part of the API (read by users). At the very least they need to be considered during upgrade/downgrade scenarios, just like "normal" APIs.

Reply:
You use the API to manage labels, but we don't apply API stability guarantees to what keys are set and what values they can have.

Reply:
In this case or in general?

Reply:
For all labels and annotations, we don't apply API stability guarantees to what keys are set and what values they can have.

Reply:
> [#4971 (comment)](https://github.com/kubernetes/enhancements/pull/4971#discussion_r1851998358) was a nit, but it makes sense to me.
> 
> (no need to check with docs folks on this - the logic about the change was to avoid implying there's a known label key)
> 
> Label keys aren't API, but we still like to note which ones are recognized and which aren't.
Thanks @sftim for clarification. i'll update the label then.